ui/StatusQ/src/StatusQ/Components/private/chart/ChartCanvas.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.0
+
+import "./Library/Library.js" as Lib
+
+Canvas {
+    id: canvas
+
+    readonly property var availablePlugins: {
+        return { datalabels: ChartDataLabels }
```

Comment:
Crosshair too?

Reply:
Crosshair is a more complicated one and it's not enough to register it in `plugins`. By default it offers only the global plugin version meaning that it will hook itself to the Chart and it can be enabled only by setting the `crosshair` options.